packages/design-system/src/styles/margin.ts

```diff
@@ -59,4 +60,9 @@ const marginStyles = marginValues.reduce((acc, value) => {
   return acc
 }, {} as Record<MarginKey, Record<MarginProperty, MarginValue>>)
 
+export const marginTheme = marginValues.reduce((acc, value) => {
```

Comment:
Why do we need an extra `marginTheme` variable? I've just noticed `paddingTheme` too. It makes me confused to have `margin`, `padding`,  `marginTheme` & `paddingTheme` variables. Can we have only `margin` & `padding`?

Reply:
The problem here is that sometimes we need to have access to those values and the current `margin` and `padding` only works within react native styles.

Maybe we just need a better naming for it instead of `marginTheme` and `paddingTheme` because you are right it is confusing 😅